src/Accordion.php

```diff
@@ -154,6 +155,20 @@ public function renderItems()
         return implode("\n", $items);
     }
 
+    /**
+     * check wether accordion is expanded
+     * This method will check if expand properties exist and determine value of this properties
+     * 
+     * @param array $item
+     * @return bool
+     */
+    private function _isAccordionExpanded($item){
```

Comment:
Methods (even private ones) aren't prefixed with underline in [Yii2 code styles](https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md#43-methods) and the brace of a method should be placed on the next line.